.ci/azure-pipelines.yml

```diff
@@ -64,11 +64,11 @@ jobs:
       matrix:
         XCode-latest:
           test_config: "ci"
-          vmImage: "macOS-13"
+          vmImage: "macOS-14"
         XCode-oldest:
           test_config: "ci"
-          vmImage: "macOS-12"
-          xcode_path: "/Applications/Xcode_13.2.1.app"
+          vmImage: "macOS-13"
+          xcode_path: "/Applications/Xcode_14.1.app"
```

Comment:
Just checking Xcode14.1 on macOS-13?  Guessing it is correct, just wanted to check

Reply:
For macOS-13 there are a few versions available, the oldest is XCode 14: https://github.com/actions/runner-images/blob/main/images/macos/macos-13-Readme.md#xcode

macOS-14 and onward will only have one major version of XCode, with a version number that is +1 the macOS version: https://github.com/actions/runner-images/issues/10703